util/dvsim/doc/_index.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "DVSim tool"
```

Comment:
@matutem @moidx 

Would you please provide an early feedback on the structure / contents of this documentation?

CC: @domrizz0

Reply:
@cfrantz fyi